src/github.rs

```diff
@@ -1216,11 +1246,21 @@ pub struct PushEvent {
     sender: User,
 }
 
+/// An event triggered by a webhook.
 #[derive(Debug)]
 pub enum Event {
+    /// A Git branch or tag is created.
     Create(CreateEvent),
+    /// A comment on an issue or PR.
+    ///
+    /// Can be:
+    /// - Regular comment on an issue or PR.
+    /// - A PR review.
+    /// - A comment on a PR review.
```

Comment:
Maybe worth a note that this is also handled in triagebot to unify these three events together.

Reply:
Sure, sounds good!